notebooks/281-depth-anything/README.md

```diff
@@ -0,0 +1,24 @@
+ Monodepth Demo
```

Comment:
```suggestion
# Monodepth Demo

Reply:
it is accidentally copied from another notebook, title changed